zio-http/src/it/scala/zhttp/IntegrationSpec.scala

```diff
@@ -0,0 +1,65 @@
+package zhttp
+
+import zhttp.http.HttpData.CompleteData
+import zhttp.http.Response.HttpResponse
+import zhttp.http._
+import zhttp.service._
+import zio._
+import zio.duration._
+import zio.test.Assertion._
+import zio.test.TestAspect.timeout
+import zio.test._
+
+object IntegrationSpec extends DefaultRunnableSpec {
+  import IntegrationSpecHelper._
+
+  def spec = suite("IntegrationSpec")(
+    ApiSpec,
+  ).provideCustomLayer(env) @@ timeout(10 seconds)
+
+  def ApiSpec = suite("ApiSpec") {
+    testM("200 ok on /") {
+      val zResponse = Client.request(baseAddr)
+      assertM(zResponse.map(_.headerRemoved("date")))(
+        equalTo(
+          HttpResponse(
+            Status.OK,
+            List(Header.custom("server", "ZIO-Http"), Header.custom("content-length", "0")),
+            CompleteData(Chunk.empty),
```

Comment:
in release/next we have a assertion named isResponse and you can check response.status res.headers etc. may be we can move those to reduce the verbosity

Reply:
also in this case if HttpData.CompleteData is renamed all tests needs to be changed however if you have a getContent function if something changes in the API you only need to change getContent